src/DetailsView/components/details-view-switcher-nav.ts

```diff
@@ -185,7 +185,6 @@ const detailsViewSwitcherNavs: {
         getSharedAssessmentFunctionalityObjects: switcher => switcher.getQuickAssessObjects(),
         getSelectedAssessmentStoreData: getQuickAssessStoreData,
         getSelectedAssessmentCardSelectionStoreData: getQuickAssessCardSelectionStoreData,
-        shouldShowQuickAssessRequirementView: true,
```

Comment:
this change is not causing any compilation errors and not breaking test cases?
Use lint check, type check, format check commands to check the errors if any

Reply:
Yes, that check is done.